packages/storage-adapter-indexeddb/src/index.js

```diff
@@ -0,0 +1,102 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+ /* eslint-env browser */
+
+import localforage from 'localforage';
+
+import {NotFoundError} from '@ciscospark/spark-core';
+
+const namespaces = new WeakMap();
+const loggers = new WeakMap();
+
+/**
+* IndexedDB adapter for spark-core storage layer
+*/
+export default class StorageAdapterIndexedDB {
+  /**
+   * @constructs {StorageAdapterIndexedDB}
+   * @param {string} basekey localforage key under which
+   * all namespaces will be stored
+   */
+  constructor() {
+    /**
+     * localforage binding
+     */
+    this.Bound = class {
+      /**
+       * @constructs {Bound}
+       * @param {string} namespace
+       * @param {Object} options
+       */
+      constructor(namespace, options) {
+        namespaces.set(this, namespace);
+        loggers.set(this, options.logger);
+      }
+
+      /**
+       * Removes the specified key
+       * @param {string} key
+       * @returns {Promise}
+       */
+      del(key) {
+        loggers.get(this).info(`indexeddb-store-adapter: deleting \`${key}\``);
+        return localforage.removeItem(key);
```

Comment:
This should probably be something more like `return localforage.removeItem(`${namespace}/${key}`);